aws/table_aws_cloudwatch_metric_data_point.go

```diff
@@ -119,6 +119,12 @@ func tableAwsCloudWatchMetricDataPoint(_ context.Context) *plugin.Table {
 				Type:        proto.ColumnType_STRING,
 				Transform:   transform.FromQual("timezone"),
 			},
+			{
+				Name:       "source_account_id",
+				Description: "Source account ID from which the metrics need to be fetched.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromQual("source_account_id"),
+			},
```

Comment:
```suggestion
			{
				Name:        "source_account_id",
				Description: "Source account ID from which the metrics need to be fetched.",
				Type:        proto.ColumnType_STRING,
				Transform:   transform.FromQual("source_account_id"),
			},
```

Not sure if spacing is off, or GitHub is just showing it incorrectly. Using go formatter may catch this as well if formatting is off?